camelot/core/profile.py

```diff
@@ -66,13 +69,16 @@ class Profile(object):
     object.
     """
     
-    def __init__( self, name, **kwargs ):
-        kwargs['name'] = name
-        for profile_field in profile_fields:
-            kwargs.setdefault( profile_field, '' )
-        for key, value in kwargs.items():
-            setattr(self, key, value )
-    
+    name: str = ''
+    dialect: str = ''
+    host: str = ''
+    port: str = ''
```

Comment:
moet dit geen int zijn ?

Reply:
Best wel, maar dan krijg je wel problemen (exceptions) met profielen die reeds "" als poort hadden. Origineel was dit een string.

Reply:
Heb het opgelost door een conversie te doen in read_profiles.